packages/tsconfig-reference/copy/id/sections/top_level.md

```diff
@@ -0,0 +1,3 @@
+### Sumber Bidang
+
+Memulai dari pilihan di dalam sumber TSConfig - banyak pilihan bagaimana berhubungan dengan projek Typescript atau Javascript yang kamu buat.
```

Comment:
Menurut saya terjemahan untuk `these options relate to how ...` itu kurang lebih `Pilihan-pilihan ini berhubungan dengan bagaimana ...`.

Kemarin kita sepakat untuk menggunakan `Anda` daripada `kamu` Mas, diskusinya bisa dilihat di sini https://github.com/microsoft/TypeScript-Website/pull/963#discussion_r475663161

Saya kurang yakin terjemahan yg pas untuk `root` itu `sumber`, mungkin dari teman-teman lain ada ide?

Reply:
Bisa merujuk ke sini 

https://twitter.com/search?q=from%3Aivanlanin%20root&src=typed_query
http://kateglo.com/?mod=glossary&op=1&phrase=root&dc=teknologiinformasi&lang=&src=&srch=Cari

Penulisan yang benar bukannya "JavaScript" dan "TypeScript" ya?

Reply:
baik mas, saya coba perbaiki lagi

Reply:
Term `you` diartikan sebagai `Anda` mas di dokumen-dokumen sebelumnya @afman42